composer.json

```diff
@@ -27,7 +27,12 @@
         "symfony/twig-bundle": "^2.3 || ^3.0"
     },
     "require-dev": {
+        "doctrine/couchdb": "^1.0.0@dev",
+        "doctrine/couchdb-odm": "^1.0.0@dev",
+        "doctrine/couchdb-odm-bundle": "^2.1.0@dev",
         "doctrine/doctrine-bundle": "^1.3",
+        "doctrine/orm": "^2.5.5",
+        "doctrine/mongodb-odm": "^1.1.1",
```

Comment:
This should be added directly in `.travis.yml` file as `composer require --dev` to not break anything.